src/extra/prisma-connection-code.md

```
@@ -0,0 +1,19 @@
+npm install prisma --dev
+
+npm i @prisma/client
+
+npx prisma init
+
+//inside lib, make a file name db.ts
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)